osCam/templates/core/home.html

```diff
@@ -44,7 +44,7 @@ <h2>cameras</h2>
                         <!-- cameras -->
                         {% if user.is_authenticated %}
                         <div class="container-fluid">
-                        <img src="{% url 'feed' %}">
+                        <img src="{% url 'feed' %}" style="border: 1px solid black">
```

Comment:
i believe there is a css file for styling

a border around the video is a good idea